src/components/GuestCheckInForm.js

```diff
@@ -50,12 +51,13 @@ class CheckInForm extends React.Component {
             id: room._id,
           },
         });
+        const data = jwt.decode(res.data);
         this.setState(cState => {
           const availableRooms = cState.availableRooms.filter(
             room => room._id !== room_id,
           );
           return {
-            loginCode: res.data.passcode,
+            loginCode: data.passcode,
             availableRooms,
             selectValue: '',
```

Comment:
there will probably be a merge conflict here. will need the `guestToken` in state to create a URL for QR code